lib/plugins/openaps.js

```diff
@@ -86,7 +87,13 @@ function init() {
 
     var info = [];
 
-    info.push({label: timeFormat(prop.status.when, sbx), value: prop.status.label});
+    var device = prop.device || 'device';
+
+    if (device.indexOf('openaps://') == 0) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Expected '===' and instead saw '=='.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1881439821/issues/source?bid=2452379&fileBranchId=3088515#l92)